manim/config.py

```diff
@@ -1,38 +1,38 @@
-"""
-config.py
----------
-
-Process the manim.cfg file and the command line arguments into a single
-config object.
-
-"""
```

Comment:
Why was this deleted? [PEP257](https://www.python.org/dev/peps/pep-0257/) clearly says

> All modules should normally have docstrings, and all functions and classes exported by a module should also have docstrings.

Please add it back.

Reply:
I missed it out while copying. Will add :)